cwms-servers/CWMS_HOME/bin/sendLogMessage

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+if [ ! -f $CWMS_EXE/bin/getProp ]
+then
+	echo " "
+	echo "Script \"$CWMS_EXE/bin/getProp\" was not found!"
+	echo "Unable to setup CWMS Enterprise Logging"
+	echo " "
+	exit 1
+fi
+
+LOGGING_HTTP_ENDPOINT=$(getProp cwms.enterprise.logging.http.url notFound cwms.properties $CWMS_HOME/config/properties)
+LOG_HOST=$(hostname)
+LOG_FACILITY='CWMS'
+CONSOLE_PRINT=true
+DEBUG=false
+
+get_log_level() {
+  local CODE=$1
+  if [ $CODE -eq 0 ]; then
+    echo "6" # INFO
+  else
+    echo "4" # WARNING
+  fi
+}
+
+send_log() {
+  local LEVEL=$1
+  local MESSAGE=$2
+  local PID=$$
+  local PROCESS_NAME=$(basename $0)
+
+  local PAYLOAD="{
+                          \"version\": \"1.1\",
+                          \"host\": \"$LOG_HOST\",
+                          \"short_message\": \"$MESSAGE\",
+                          \"full_message\": \"$MESSAGE\",
+                          \"timestamp\": $(date +%s),
+                          \"level\": $LEVEL,
+                          \"facility\": \"$LOG_FACILITY\",
+                          \"_pid\": \"$PID\",
+                          \"_process_name\": \"$PROCESS_NAME\"
+                      }"
+  #echo $PAYLOAD
+  curl -X POST -H 'Content-Type: application/json' -s -d "$PAYLOAD" $LOGGING_HTTP_ENDPOINT
```

Comment:
this is https?

Reply:
depends on how the cwms.properties is configured